src/components/ui/AddLocationPin.js

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components/macro'
+
+import PinSvg from './AddLocationPin.svg'
+
+// TODO: use icon from a library
```

Comment:
future PR?

Reply:
Yes, just wanted to get this code merged in. Shouldn’t make PRs that are too large, repeating a GLW mistake here LOL